omod-1.8/src/main/java/org/openmrs/module/webservices/rest/web/v1_0/search/openmrs1_8/PatientByIdentifierSearchHandler1_8.java

```diff
@@ -43,14 +48,23 @@ public SearchConfig getSearchConfig() {
 	public PageableResult search(RequestContext context) throws ResponseException {
 		
 		String identifier = context.getRequest().getParameter("identifier");
+		String searchType = context.getParameter("searchType");
 		
 		if (StringUtils.isNotBlank(identifier)) {
-			List<Patient> patients = Context.getPatientService().getPatients(null, identifier, null, true);
-			if (patients != null && patients.size() > 0) {
-				return new NeedsPaging<Patient>(patients, context);
+			if ("start".equals(searchType)) {
+				List<Patient> patients = restHelperService.findPatientsByIdentifierStartingWith(identifier, false);
```

Comment:
I did write comment above that there is no such method as Context.getIncludeAll()